instaloader/structures.py

```diff
@@ -408,6 +408,21 @@ def get_sidecar_nodes(self, start=0, end=-1) -> Iterator[PostSidecarNode]:
                     yield PostSidecarNode(is_video=is_video, display_url=display_url,
                                           video_url=node['video_url'] if is_video else None)
 
+    def get_translated_caption(self) -> Optional[str]:
+        """Retrieve translation of the caption.
+
+        .. versionadded:: 4.10"""
```

Comment:
We released `4.10`, this should go into `4.11` then.

```suggestion
        .. versionadded:: 4.11
```